### PR TITLE
Fix align issue and update cloud page

### LINKED ISSUE #539: u-align--center missing value
## Summary

- hX and p tags with class "u-align--center" missing value which causes the texts alignments issues. 
- Logos list broken on cluod page

## Current and expected result
[Current live environment and local test environment with the latest vanilla framework are broken]

## Screenshot
URL: https://cn.ubuntu.com/server
image

URL: https://cn.ubuntu.com/cloud
image

### PR DESCRIPTION
**Pull request by zprood**
## Done

- Fix server page text alignment issue
- Update the cloud page because the logos list broken and the content is out of date
- Created subpage for public cloud 

## Issue / Card
Fix #539 

## Screenshots
image

### CODE REVIEW

File: templates/cloud/index.html [hunk 1]
```diff
@@ -1,526 +1,241 @@
...
-  <div class="row">
-    <div class="col-8">
-      <h2>立即安装OpenStack</h2>
-      <p>利用<a href="https://conjure-up.io/" class="p-link--external">conjure-up</a>工具，可在单个笔记本电脑上使用LXD容器快速部署OpenStack，这种方式非常适合首次使用者或开发者。您也可以使用相同的工具搭配<a href="https://maas.io/" class="p-link--external">MAAS</a>，在裸机机架服务器、OpenStack 或 VMware上进行运行。</p>
-      <p><a class="p-link--external" href="https://www.ubuntu.com/download/cloud">安装OpenStack</a></p>
-    </div>
+  <div class="u-fixed-width">
+    <p><a href="https://ubuntu.com/openstack" class="p-link--external">Canonical的Charmed OpenStack</a>通过简化OpenStack的部署和提供自动化运维来解决这个问题，从而显着降低私有云的总拥有成本 (TCO)。</p>
+    <p>
+      <a href="https://ubuntu.com/openstack" class="p-link--external">获取一个受支持的，托管的，电信运营商级的私有云&nbsp;›</a>
```
Comment (anthonydillon) [inline templates/cloud/index.html:70]:
```suggestion
      <a href="https://ubuntu.com/openstack" class="p-link--external">获取一个受支持的，托管的，电信运营商级的私有云</a>
```

File: templates/cloud/index.html [hunk 2]
```diff
@@ -1,526 +1,241 @@
...
+      <p><b>缺点：</b></p>
+      <ul class="p-list">
+        <li class="p-list__item is-crossed">虽然公有云的总体成本看起来很有吸引力，但隐藏的附属成本通常使其成为云架构的昂贵选择。</li>
       </ul>
     </div>
   </div>
+  <div class="u-fixed-width">
+    <p>主要的公有云提供商包括亚马逊（AWS）、微软Azure、Google Cloud Platform、甲骨文和IBM Cloud。</p>
+    <p>
+      <a href="/public-cloud">在公有云上自信地使用Ubuntu来运行您的工作负载&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/cloud/index.html:112]:
```suggestion
      <a href="/cloud/public-cloud">在公有云上自信地使用Ubuntu来运行您的工作负载&nbsp;&rsaquo;</a>
```

File: templates/cloud/index.html [hunk 3]
```diff
@@ -1,526 +1,241 @@
...
-        <li class="p-inline-images__item p-inline-images__item--smaller">
-          <img src="https://assets.ubuntu.com/v1/507a0979-Voidbridge.jpg" alt="Voidbridge">
-        </li>
-        <li class="p-inline-images__item p-inline-images__item--smaller">
-          <img src="https://assets.ubuntu.com/v1/2e38f398-uct.svg" alt="UCT">
-        </li>
+    <div class="col-5">
+      <p><b>缺点：</b></p>
+      <ul class="p-list">
+        <li class="p-list__item is-crossed">虽然公有云的总体成本看起来很有吸引力，但隐藏的附属成本通常使其成为云架构的昂贵选择。</li>
```
Comment (anthonydillon) [inline templates/cloud/index.html:105]: Can you add the `is-crossed`?

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Great work @zprood this looks great. Only a few small suggestions in the code.

File: templates/cloud/index.html (hunk 1 above)
Comment (zprood) [reply templates/cloud/index.html]: @anthonydillon Thanks a lot. Removed

File: templates/cloud/index.html (hunk 2 above)
Comment (zprood) [reply templates/cloud/index.html]: Added "/cloud"

File: templates/cloud/index.html (hunk 3 above)
Comment (zprood) [reply templates/cloud/index.html:105]: Yes, it's missing in the style.scss, added.

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**